tox.ini

```diff
@@ -50,6 +50,7 @@ commands =
 description = build json schemas in docs/
 deps =
     pyyaml
+    git+https://github.com/PietroPasotti/interface-tester-pytest
```

Comment:
Are you planning to park this dependency under the Canonical organization on GitHub and have Canonical devs maintain it in the future? I am a slightly wary towards becoming reliant on a package hosted in your personal GitHub especially if it is expected that all the schemas will transition to this new pydantic model from the JSON. Is this just the same thing as the schema base that was deleted in this pull request?

Reply:
Yes, I created it under my personal namespace by mistake and will move it under Canonical before releasing it as pypi package and hooking up the necessary plumbing to do so on each release.
I'd do that immediately if it weren't that it's a bit of work given that if you transfer a repo to Canonical you lose ownership and it can take ages for IS to give it back to you (still waiting for traefik-rock after 3 months...)

But good point, good to keep that in mind.

Reply:
And yes, the model base class is exactly the one that was removed from the repo in this PR (modulo the docstring).

Reply:
> (still waiting for traefik-rock after 3 months...)

Makes sense provided these issues. Generally if I get impatient, I'll just create a new repository under the Canonical org and then reconfigure the upstream remote on my local machine. I did this with the NFS operators. It's not great, but at least you are not waiting 3 months :sweat_smile: